config/locales/es.yml

```diff
@@ -2505,6 +2505,9 @@ es:
             has_out_of_state_w2: recibiste ingresos en otro estado en el %{filing_year}
             has_yonkers_income: viviste, recibiste ingresos o mantuviste un hogar en Yonkers en 2024.
             married_filing_separately: estás presentando una declaración de impuestos "casado, declarando por separado".
+            spouse_nra_html: |
```

Comment:
i think there's spanish copy in the story!

Reply:
![Screenshot 2025-03-20 at 9 24 28 AM](https://github.com/user-attachments/assets/97d93ad9-c196-471f-b3c7-a634b3bf1810)

This is the spanish copy right now.

Reply:
ooh can we ask for a translation? Or ask why it looks this way?